src/OrleansServiceBus/Providers/EventDataGeneratorStreamProvider/EventDataGeneratorStreamProvider.cs

```diff
@@ -64,6 +99,17 @@ public void PopulateDataGeneratingConfigFromProviderConfig(IProviderConfiguratio
         public void WriteDataGeneratingConfig(Dictionary<string, string> properties)
         {
             properties.Add(StreamDataGeneratorTypeName, this.StreamDataGeneratorType.AssemblyQualifiedName);
+            properties.Add(EventHubPartitionRangeStartName, this.EventHubPartitionRangeStart.ToString());
+            properties.Add(EventHubPartitionRangeEndName, this.EventHubPartitionRangeEnd.ToString());
+        }
+
+        public static string[] GenerateEventHubPartitions(int rangeStart, int rangeEnd)
```

Comment:
Unclear why we need this level of control.  Isn't it sufficient to simply declare how many partitions we want, rather than having a start and end?